mlflow/store/model_registry/abstract_store.py

```diff
@@ -140,8 +140,15 @@ def delete_registered_model_tag(self, name, key):
 
     @abstractmethod
     def create_model_version(
-        self, name, source, run_id=None, tags=None, run_link=None, description=None
-    ):
+        self,
+        name,
+        source,
+        run_id=None,
+        tags=None,
+        run_link=None,
+        description=None,
+        local_model_path=None,
+    ):  # pylint: disable=unused-argument
```

Comment:
Can we remove `# pylint: disable=unused-argument`? This is an abstract method. pylint should ignore `unused-argument`.